wren-ui/src/apollo/server/schema.ts

```diff
@@ -684,6 +684,7 @@ export const typeDefs = gql`
     error: Error
     description: String
     chartSchema: JSON
+    adjustment: Boolean
```

Comment:
@andreashimin where do you do a fallback ? I mean for response not adjusted or older responses not having this value, will this value default to `false` ?

Reply:
there's no fallback for the value since we cannot know the existed chart schemas were adjusted or not.

The value will only appear `true` or `null`.

so if null the regenerate button will not show. But for new asking, it should be appear if the encounter the scenario.